tensorflow_graphics/projects/pointnet/train.py

```diff
@@ -128,42 +126,32 @@ def evaluate():
   return evaluate.best_accuracy
 
 
-# ------------------------------------------------------------------------------
-# ------------------------------------------------------------------------------
-# ------------------------------------------------------------------------------
+if not FLAGS.dryrun:
```

Comment:
Is this still needed?
can't we just remove the dryrun mechanism altogether now?

Reply:
I usually get in trouble when I change too many things in one PR... ;)

Reply:
sure, but this is pretty small :)
just go ahead and remove it, it is now useless (thanks to TFDS)